make dev-setup fails to run properly

I am trying to set up the dev environment for the first time.  I am sure I just have something set up incorrectly, but the instructions don't give me any clues what I've done wrong.  Here is what I'm seeing for the output:

```
docker-compose run --rm php php artisan migrate

   Illuminate\Database\QueryException 

  SQLSTATE[HY000] [2002] Connection timed out (SQL: select * from information_schema.tables where table_schema = votelocal and table_name = migrations and table_type = 'BASE TABLE')

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:671
    667|         // If an exception occurs when attempting to run a query, we'll format the error
    668|         // message to include the bindings with SQL, which will make this exception a
    669|         // lot more helpful to the developer instead of just the database's errors.
    670|         catch (Exception $e) {
  > 671|             throw new QueryException(
    672|                 $query, $this->prepareBindings($bindings), $e
    673|             );
    674|         }
    675| 

      +36 vendor frames 
  37  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
make: *** [Makefile:39: db-migrate] Error 1

```

--- a/DOCKER.md
+++ b/DOCKER.md
@@ -10,7 +10,7 @@ Copy the two configuration files into place (note: `.db.env` is not required for
 
 If you're looking to test out the **Twilio** and/or **Twitter** integrations locally, you will need to configure the appropriate sections in your new `.env` file.
 
-Laravel requires `storage` and `bootstrap/cache` directories to be writeable by `www-data`. Depending on your environment there may be several ways to accomplish this. One method is:
+Laravel requires `storage` and `bootstrap/cache` directories to be writeable by `www-data`. Depending on your environment this may already be the case. If not, there are several ways to accomplish this. One method is:
 ```
 $ sudo chgrp -R www-data storage bootstrap/cache
 $ chmod -R g+w storage bootstrap/cache

--- a/DOCKER.md
+++ b/DOCKER.md
@@ -10,6 +10,12 @@ Copy the two configuration files into place (note: `.db.env` is not required for
 
 If you're looking to test out the **Twilio** and/or **Twitter** integrations locally, you will need to configure the appropriate sections in your new `.env` file.
 
+Laravel requires `storage` and `bootstrap/cache` directories to be writeable by `www-data`. Depending on your environment there may be several ways to accomplish this. One method is:
+```
+$ sudo chgrp -R www-data storage bootstrap/cache
+$ chmod -R g+w storage bootstrap/cache
+```
+
 ## Quick Start
 
 First start your containers.

